src/noMissingVariablesInEnvFiles.native.test.ts

```diff
@@ -0,0 +1,56 @@
+import fs from 'fs'
+
+function loadEnvVariables(filePath: string) {
+  if (!fs.existsSync(filePath)) {
+    throw new Error(`File not found: ${filePath}`)
+  }
+  const content = fs.readFileSync(filePath, 'utf-8')
+  const lines = content.split('\n')
+
+  const envVars: string[] = []
+  lines.forEach((line) => {
+    line = line.trim()
+    if (line && !line.startsWith('#')) {
+      const [key] = line.split('=')
+      if (key) envVars.push(key.trim())
+    }
+  })
+
+  return envVars
+}
+
+function compareEnvFiles(envFiles: string[]) {
+  const envData: Record<string, string[]> = {}
```

Comment:
j'ai l'impression qu'il pourrait etre intéressant d'avoir un `Set` en valeur

```suggestion
  const envData: Record<string, Set<string>> = {}
```

Reply:
Si on avait la méthode Set.union et Set.symetricDifference, carrément, mais ce n'est pas le cas en NodeJS, donc c'est + simple avec un array :/